sampita/welcome-to-nashville
# Added styles

# Description

Added styling to application


## Type of change

Please delete options that are not relevant.
- [x ] New feature (non-breaking change which adds functionality)

# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout added-styles`
4. Check to see if application is styled with Nashville header image and clean styling.

# Checklist:
